index.js

```diff
@@ -1 +1,22 @@
-// phew… not a lot going on here. Please add some code!
+const bookMarkToggle = document.querySelector('[data-js="bookMark"]');
+
```

Comment:
Better practice would be, to locate all variables, which are declared with the `querySelector` here on the top of the page. (Line 7 to 11 also here above the rest of the code)